### DISCUSSION TITLE
Welcome to Prot1ntelligence Discussions!

### DISCUSSION THREAD

**Discussion by C3n7ral051nt4g3ncy** [C3n7ral051nt4g3ncy/Prot1ntelligence#2] [General]
## 👋 Welcome to Prot1ntelligence Discussions
<br>
<img width="1013" src="https://user-images.githubusercontent.com/104733166/177401455-f208630f-f6e7-4cd2-9363-14ecbcade716.png">
<br>

**Prot1ntelligence** is my first Python Script. 

I have activated Discussions on my repository, to connect with other members of the GitHub community, and to be closer to followers or people who may need some help! So don't hesitate to:
  * Ask questions ❓
  * Share your thoughts and ideas  💡 
  * Share any issues you may have  😓
  * We are all in the same community and should all support each other 🫶

**Comment by EdenBlue** [comment]
Hi
I am discovering your tool which seems very interesting.
However, can you please explain to me the link (in the cases of Bravo and Charly) between the typed email and the found web links? What can we deduce from this? I tested it on myself and I don't understand 😅 Thanks for your help.

**Reply by C3n7ral051nt4g3ncy** [reply]
Hi @EdenBlue 

The `Bravo` + `Charly` modules are added features to make life a little easier , it means you can run a search within the tool without needing to open your browser.

- `Bravo`: Uses the `Googlesearch-python` lib, basically like running a google search in your browser, but you can do it from the terminal. *If your email is somewhere out there (open source), then you will get results as links. Sometimes even when there is no information, some links come back and are inaccurate , so you need to check each link (select link --> click right  --> open selection as url) .

- `Charly`: This is the dark web search option which uses `Ahmia`, you can choose to have results in terminal, or browser auto-opening which is also cool.
